Himanshuagnihotri.java

```diff
@@ -0,0 +1,27 @@
+import java.io.*;
+import java.util.*;
+import java.util.regex.*;
+class Himanshu
+{
+  public static void main(String args[])
+ {
+    Scanner sc=new Scanner(System.in);
+    int n,i;
+    ArrayList<Integer>res = new ArrayList<>();
+    String text=sc.nextLine();
+    String a[]=new  String[100];
+    n=Integer.parseInt(sc.nextLine());
+    	for(i=0;i<n;i++)
```

Comment:
Please use proper naming convention or meaningful variable names while declaring variables. And create a separate method to get the output.